tests/BearSerializationTest.py

```diff
@@ -0,0 +1,34 @@
+import itertools
+import json
+import os
+import json
+import unittest
+
+from pyprint.NullPrinter import NullPrinter
+
+from coalib.bears.BEAR_KIND import BEAR_KIND
+from coalib.collecting.Collectors import collect_bears
+from coalib.output.JSONEncoder import create_json_encoder
+from coalib.output.printers.LogPrinter import LogPrinter
+
+
+class BearSerializationTest(unittest.TestCase):
+    """
+    Test to make sure all bears are serializable.
+    """
+
+    def setUp(self):
+        printer = LogPrinter(NullPrinter())
+        local_bears, global_bears = collect_bears(
+            [os.path.abspath("bears")],
+            ["**"],
+            [BEAR_KIND.LOCAL, BEAR_KIND.GLOBAL],
+            printer,
+            warn_if_unused_glob=False)
+        self.bears = list(itertools.chain(local_bears, global_bears))
+        self.JSONEncoder = create_json_encoder(use_relpath=False)
+
+    def test(self):
+        json.dumps(self.bears, cls=self.JSONEncoder)
+        # If no exception is thrown then the test passes
```

Comment:
@arafsheikh So IIUC for non-serialisation case you are depending on an exception to be raised? Can we rather emulate a bear that is non serialisable and test for that too? Otherwise, at least catch the particular exception which is raised for non serialisable bears and assert for errors neatly. 
I guess it would be little near as it won't just populate the console with tracebacks. Thoughts?